Rules.md

```diff
@@ -95,3 +95,11 @@ Display error when views are ambiguous.
 |   Default Rule  |  `false` |
 
 Check View as: set as a device specified by view_as_device_rule config.
+
+## DuplicateIdRule
```

Comment:
I prefer to name `DuplicateIDRule` by following to naming guideline.

Reply:
Sometime I use `Id`, some time I use `ID` but never `iD`!
I know this could be an exception but for my knowledge I want to know where we can find this naming rule (swift naming convention? this project in particular?)
because upper camelcase of "Duplicate ID Rule" is `DuplicateIdRule`